File: srm178/simple_calculator.py
class SimpleCalculator:
	def calculate(self, input):
		ops = ['-', '+', '/', '*']
		op = ''
		nums = []
		num = ''
		for i in range(len(input)):
			if input[i] in ops:
				op = input[i]
				nums.append(int(num))
				num = ''
			else: num += input[i]
		nums.append(int(num))
				
		return eval(str(nums[0]) + op + str(nums[1]))

File: srm178/SimpleCalculator.java
import java.lang.*;

public class SimpleCalculator {

	public int calculate(String input) {
		String ops = "-+/*";
		String[] nums = new String[2];
		char op = ' ';
		String num = "";
		int res = 0;
		for (int i = 0; i < input.length(); i++) {
			if (input.charAt(i) == '-' || input.charAt(i) == '+' || input.charAt(i) == '*' || input.charAt(i) == '/') {
				op = input.charAt(i);
				nums[0] = num;
				num = "";
			} else { num += input.charAt(i); }
		}
		nums[1] = num;
		
		switch (op) {
			case '-': res = Integer.parseInt(nums[0]) - Integer.parseInt(nums[1]);
					  break;
					   
			case '+': res = Integer.parseInt(nums[0]) + Integer.parseInt(nums[1]);
					  break;
			
			case '*': res = Integer.parseInt(nums[0]) * Integer.parseInt(nums[1]);
					  break;
			
			case '/': res = Integer.parseInt(nums[0]) / Integer.parseInt(nums[1]);
					  break;
		}
		return res;
	}
}